L/LAPACK/LAPACK32/build_tarballs.jl

```diff
@@ -10,6 +10,7 @@ products = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               clang_use_lld=false, julia_compat="1.9", preferred_gcc_version=v"6")
+               clang_use_lld=false, julia_compat="1.9", preferred_gcc_version=v"8")
```

Comment:
For libgfortran4 the "preferred gcc version" is irrelevant: gcc 7 will always be used.

Reply:
Thanks.